.gitignore

```diff
@@ -47,4 +47,9 @@ dist
 # Environment variables
 .env
 
-*.sqlite
\ No newline at end of file
+*.sqlite
+# User Table Mock
+mock
+
+# Prettier confi
+.prettierrc
```

Comment:
Why is this in the gitignore please remove

Reply:
I am using this for my vscode settings to enable code formatting,

Reply:
My point is that this change will affect everybody else's configuration please remove this line

Reply:
For more insight go a research what a `.prettierrc` file does